tests/testJointcal.py

```diff
@@ -10,7 +10,7 @@
 import eups
 
 from lsst.afw import geom, coord, table
-from lsst.meas.astrom import astrometry
+from lsst.meas import astrom
```

Comment:
I suggest `from lsst.meas.astrometry import LoadAstrometryNetObjectsTask`
I worry that it's hard for a user to tell what is brought in when you import all of lsst.meas (it's not a usage I've ever seen in our code before). Note that you probably don't need the config